x-pack/plugins/secops/public/components/page/hosts/events_table/index.tsx

```diff
@@ -80,36 +142,36 @@ const getEventsColumns = (startDate: number, endDate: number) => [
     sortable: true,
     truncateText: true,
     hideForMobile: true,
-    render: (item: EventItem) => (
-      <ProviderContainer>{getOr('--', 'event.type', item)}</ProviderContainer>
-    ),
+    render: ({ event }: { event: Ecs }) => <>{getOr('--', 'event.type', event)}</>,
   },
   {
     name: 'Source',
     truncateText: true,
-    render: (item: EventItem) => (
+    render: ({ event }: { event: Ecs }) => (
       <>
-        {getOr('--', 'source.ip', item).slice(0, 12)} : {getOr('--', 'source.port', item)}
+        {has('source.ip', event) ? getOr('--', 'source.ip', event).slice(0, 12) : '--'} :{' '}
+        {getOr('--', 'source.port', event)}
```

Comment:
At one point I made a `EMPTY_VALUE` for the renderers and played with a curry that cut down on syntactical noise here:

https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/components/timeline/body/renderers/plain_column_renderer.tsx#L20

At some point we could move that up higher as a global getOrEmpty curry above this and then everything would be code like so

```ts
getOrEmpty('source.port', event)
```

Then if we change out the `--` for some reason it would be just one spot to change. No changes asked for this PR.